scripts/all-tests.sh

```diff
@@ -110,7 +114,32 @@ END
                 paver coverage
                 ;;
         esac
+        exit $EXIT
+        ;;
+
+    "lms-unit")
+        case "$SHARD" in
+            "1")
+                paver test_system -s lms --extra_args="--attr='shard_1' --with-flaky" --cov_args="-p" || { EXIT=1; }
+                ;;
+            "2")
+                paver test_system -s lms --extra_args="--attr='shard_1=False' --with-flaky" --cov_args="-p" || { EXIT=1; }
+                ;;
+            *)
+                paver test_system -s lms --extra_args="--with-flaky" --cov_args="-p" || { EXIT=1; }
+                ;;
+        esac
+        exit $EXIT
+        ;;
+
+    "cms-unit")
+        paver test_system -s cms --extra_args="--with-flaky" --cov_args="-p" || { EXIT=1; }
```

Comment:
If we were to add a case statement under here with a catch-all now, would that help if we later want to break these out into shards also?